packages/contracts/contracts/core/DAO.sol

```diff
@@ -113,6 +132,21 @@ contract DAO is
         _registerInterface(type(IDAO).interfaceId);
         _registerInterface(type(IERC1271).interfaceId);
         _registerInterface(type(IEIP4824).interfaceId);
+        _registerInterface(type(IERC721ReceiverUpgradeable).interfaceId);
+        _registerInterface(type(IERC1155ReceiverUpgradeable).interfaceId);
+
+        _registerCallback(
```

Comment:
1. I'd not include comments such as `0x150b7a02 => onERC721Received(address,address,uint256,bytes)`...

Reply:
2. Maybe a function: `_registerTokenInterfacesAndCallbacks` ?